### PR TITLE
Add recent pipelines releases to sync config

### PR DESCRIPTION
**Pull request by lbernick**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- n/a Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: sync/config/pipelines.yaml [hunk 1]
```diff
@@ -26,6 +26,26 @@ archive: https://github.com/tektoncd/pipeline/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+- name: release-v0.44.x
+  displayName: v0.44.x
```
Comment (AlanGreene) [inline sync/config/pipelines.yaml:30]:
```suggestion
  displayName: v0.44.x LTS
```

**Review by AlanGreene — CHANGES_REQUESTED** [review]
Thanks @lbernick, just 1 small change to mark the most recent release as LTS (similar to v0.41.x)

File: sync/config/pipelines.yaml (hunk 1 above)
Comment (lbernick) [reply sync/config/pipelines.yaml]: done!

**Review by AlanGreene — APPROVED** [review]

**Review by geriom — COMMENTED** [review]
/lgtm

**Pull request merged**